spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

```diff
@@ -175,9 +175,13 @@ Please ensure that the endpoint URI contains the path (e.g. `/api/v2/metrics/ing
 	  metrics:
 	    export:
 	      dynatrace:
-	        # uri: "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest" for managed deployments.
-	        uri: "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest"
-	        api-token: "YOUR_TOKEN" # should be read from a secure source and not hard-coded.
+	        # uri: "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest" # for SaaS
+	        # uri: "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest"    # for managed deployments
+	        # or use the Spring placeholder notation to read it from an environment variable:
```

Comment:
I don't think this is warranted. Any property can be used with such a placeholder.

Reply:
In some user's environments this particular property will already be populated. I didn't want to lay out in which cases this applies since that's subject to change and likely too detailed for these docs. How about this?
```
# If the DT_METRICS_INGEST_URL and DT_METRICS_INGEST_API_TOKEN are populated in your environment, you can use these with the Spring placeholder notation:
uri: ${DT_METRICS_INGEST_URL}
# should be read from a secure source and not hard-coded.
api-token: ${DT_METRICS_INGEST_API_TOKEN}
```

Reply:
> should be read from a secure source and not hard-coded

I don't think this is necessary. There are numerous properties mentioned elsewhere in the documentation where this is the case and we don't have such a comment for each of them.